src/app/modules/account/sagas.ts

```diff
@@ -71,15 +73,26 @@ export function* handleGetTransactions(): SagaIterator {
     const paymentNodes: Array<Yielded<typeof nodeLib.getNodeInfo>> = yield all(
       paymentNodeIds.map(id => call(safeGetNodeInfo, nodeLib, id))
     );
-    const payments = paymentsRes.payments.map(p => {
-      const nodeResponse = paymentNodes.find(
-        n => p.path[p.path.length - 1] === n.node.pub_key
-      );
-      return {
-        ...p,
-        to: (nodeResponse as any).node,
-      };
-    });
+    const payments = paymentsRes.payments
+      .map(p => {
+        const nodeResponse = paymentNodes.find(
+          n => p.path[p.path.length - 1] === n.node.pub_key
+        );
+        return {
+          ...p,
+          to: (nodeResponse as any).node,
+        };
+      })
+      .map(p => {
+        if (!p.value_sat && !p.value_msat) {
+          return {
+            ...p,
+            value_sat: p.value,
+            value_msat: fromBaseToUnit(p.value, Denomination.MILLIBITCOIN),
```

Comment:
msat is not millibitcoin, it's millisats, where each is one one-thousandth of a satoshi. We're not using it anywhere, but this would be an incorrect number.

As far as I can tell, the simplest but best way to convert value to this would be to do `${p.value}000`.

Reply:
oops. fixed